.github/actions/tests/scala_test/action.yml

```diff
@@ -133,6 +136,35 @@ runs:
       with:
         cache_version: 4
 
+    - name: Set Daml package versions
```

Comment:
This is not as precise as we probably should be: It relies on the version shipped to mainnet, not the versions vetted on mainnet. The latter is a bit of a mess since we also need a splitwell version which is not in the packageConfig, definitely solvable but annoying. So for now I'd suggest we keep it as is and open a separate issue for making the version we test against it more precise. This still gives us a two week compatibility window at a minimum which is much better than what we had so far.